pkg/iptables/iptables.go

```diff
@@ -344,11 +290,31 @@ func deleteRuleRaw(iptablesCmd string, table Table, rule ...string) (string, err
 	}
 
 	// Delete rule
-	cmd = exec.Command(iptablesCmd, append(append(args, "-D"), rule...)...)
-	out, err = cmd.CombinedOutput()
+	out, err = runIptables(iptablesCmd, table, append([]string{"-D"}, rule...)...)
 	if err != nil {
 		return string(out), err
 	}
 
 	return string(out), nil
 }
+
+// Run iptables with lock
+func runIptables(iptablesCmd string, table Table, args ...string) ([]byte, error) {
```

Comment:
I think runWithLock name is Intuitive.

Reply:
I see where you're coming from, but I disagree. I view this as a common way of running `iptables`, it just happens that it adds the lock-related command-line options. After all, `iptables` always uses the lock anyway, the options are there to make it wait for the lock rather than aborting immediately.

Reply:
Ok. Thanks.